app/viewModels/houseConsignment/index/items/packages/AddAnotherPackageViewModel.scala

```diff
@@ -0,0 +1,111 @@
+
+  def title(houseConsignmentIndex: Index, itemIndex: Index)(implicit messages: Messages): String =
+    messages(s"$prefix.$emptyOrSingularOrPlural.title", count, itemIndex.display, houseConsignmentIndex.display)
+
+  def heading(houseConsignmentIndex: Index, itemIndex: Index)(implicit messages: Messages): String =
+    messages(s"$prefix.$emptyOrSingularOrPlural.heading", count, itemIndex.display, houseConsignmentIndex.display)
+
+  def legend(houseConsignmentIndex: Index, itemIndex: Index)(implicit messages: Messages): String =
+    if (count <= 0) {
+      messages(s"$prefix.empty.label", count, itemIndex.display, houseConsignmentIndex.display)
+    } else {
+      messages(s"$prefix.label", count, itemIndex.display, houseConsignmentIndex.display)
+    }
+
+  def maxLimitLabel(houseConsignment: Index, itemIndex: Index)(implicit messages: Messages): String =
+    messages(s"$prefix.maxLimit.label", count, itemIndex.display, houseConsignment.display)
+
+}
+
+object AddAnotherPackageViewModel {
+
+  class AddAnotherPackageViewModelProvider {
+
+    def apply(userAnswers: UserAnswers, arrivalId: ArrivalId, houseConsignmentIndex: Index, itemIndex: Index, mode: Mode): AddAnotherPackageViewModel = {
+
+      val array = userAnswers.get(PackagingListSection(houseConsignmentIndex, itemIndex))
+
+      val listItems = array
+        .getOrElse(JsArray())
+        .value
+        .zipWithIndex
+        .flatMap {
+          case (_, index) =>
+            val packageIndex = Index(index)
+
+            userAnswers
+              .get(PackageTypePage(houseConsignmentIndex, itemIndex, packageIndex))
+              .flatMap(
+                packageType =>
+                  userAnswers
+                    .get(NumberOfPackagesPage(houseConsignmentIndex, itemIndex, packageIndex))
+                    .map(
+                      quantity => s"$quantity * $packageType"
+                    )
+              )
+              .orElse(
+                userAnswers
+                  .get(PackageTypePage(houseConsignmentIndex, itemIndex, packageIndex))
+                  .map(
+                    packageType => s"$packageType"
+                  )
+              )
+              .map {
+                name =>
+                  ListItem(
+                    name = name,
+                    changeUrl = None,
+                    removeUrl = Some(
+                      controllers.houseConsignment.index.items.packages.routes.RemovePackageTypeYesNoController
+                        .onPageLoad(arrivalId, houseConsignmentIndex, itemIndex, packageIndex, mode)
+                        .url
+                    )
+                  )
+              }
+        }
+        .toSeq
+
+      new AddAnotherPackageViewModel(
+        listItems,
+        onSubmitCall =
+          controllers.houseConsignment.index.items.packages.routes.AddAnotherPackageController.onSubmit(arrivalId, houseConsignmentIndex, itemIndex, mode),
```

Comment:
Consider adding `import controllers.houseConsignment.index.items.packages.routes` to imports